doc/source/advanced.rst

```diff
@@ -231,16 +228,31 @@ Advanced indexing with hierarchical index
 -----------------------------------------
 
 Syntactically integrating ``MultiIndex`` in advanced indexing with ``.loc`` is a
-bit challenging, but we've made every effort to do so. For example the
-following works as you would expect:
+bit challenging, but we've made every effort to do so. In general, MultiIndex
+keys take the form of tuples. For example, the following works as you would expect:
 
 .. ipython:: python
 
    df = df.T
    df
-   df.loc['bar']
    df.loc['bar', 'two']
 
+If you also want to index a specific column with ``.loc``, you have to use
```

Comment:
This is not very clear. You must use a tuple is more explicit.

Reply:
OK!